submissions/fluidic-power/fluidic-power.md

```diff
@@ -2,57 +2,57 @@
 
-Then flame_Sla mentioned that he once measured the Fluid Manager's time usage accurately on the world's biggest megabase, and saw that it only used `~0.4ms` for a [40KSPM 60UPS megabase](https://www.reddit.com/r/factorio/comments/nmxayx/new_ups_record_40k_spm_60_ups_no_mods_details_in/)! That means only 2.5% of the update time was spent on fluid calculations! The way he achieved this is by completely _eliminating_ the Electric Network from the time usage, which he achieved by changing all machines to [run on wood as fuel](https://discord.com/channels/579345487371567105/579346716243787782/825170120162148383). Due to how Fluidic Power works I cannot easily eliminate the effect of the Electric Network the same way without the Fluid Manager's load reducing, but I can do it the other way around! I can remove all the _fluid_ components, and replace them with EEIs to supply or drain the power. This will cleanly elimiate all added fluid computations without removing the strenuous load on the Electric Network - which is very important! In order to record an accurate and comparable time usage metric I also attempted to force Factorio to [run on a single core](https://www.reddit.com/r/technicalfactorio/comments/mead38/how_to_turn_off_multithreading_to_get_more_useful/). So I ran I quick script to replace the hidden fluid components in the power poles and the result are as follows:
+Then *flame_Sla* mentioned that he once measured the Fluid Manager's time usage accurately on the world's biggest megabase, and saw that it only used `~0.4ms` for a [40KSPM 60UPS megabase](https://www.reddit.com/r/factorio/comments/nmxayx/new_ups_record_40k_spm_60_ups_no_mods_details_in/)! That means only 2.5% of the update time was spent on fluid calculations! The way he achieved this is by completely _eliminating_ the Electric Network from the time usage, which he achieved by changing all machines to [run on wood as fuel](https://discord.com/channels/579345487371567105/579346716243787782/825170120162148383). Due to how Fluidic Power works, I cannot easily eliminate the effect of the Electric Network the same way without also reducing the Fluid Manager's load, but I can do it the other way around! I can remove all the _fluid_ components, and replace them with EEIs to supply or drain the power. This will cleanly elimiate all added fluid computations without removing the strenuous load on the Electric Network - which is very important! In order to record an accurate and comparable time usage metric I also attempted to force Factorio to [run on a single core](https://www.reddit.com/r/technicalfactorio/comments/mead38/how_to_turn_off_multithreading_to_get_more_useful/). So I ran a quick script to replace the power poles' hidden fluid components and the result are as follows:
```

Comment:
```suggestion
*flame_Sla* mentioned that he once measured the Fluid Manager's time usage on the world's biggest megabase, and that it only used `~0.4ms` for a [40KSPM 60UPS megabase](https://www.reddit.com/r/factorio/comments/nmxayx/new_ups_record_40k_spm_60_ups_no_mods_details_in/)! That means only 2.5% of the update time was spent on fluid calculations! The way he achieved this is by completely _eliminating_ the Electric Network from the time usage, which he achieved by changing all machines to [run on wood as fuel](https://discord.com/channels/579345487371567105/579346716243787782/825170120162148383). Due to how Fluidic Power works, I cannot easily eliminate the effect of the Electric Network the same way without also reducing the Fluid Manager's load, but I can do it the other way around! I can remove all the _fluid_ components, and replace them with EEIs that supply or drain the power. This will cleanly eliminate all added fluid computations without removing the strenuous load on the Electric Network - which is very important! In order to record an accurate and comparable time usage metric I also attempted to force Factorio to [run on a single core](https://www.reddit.com/r/technicalfactorio/comments/mead38/how_to_turn_off_multithreading_to_get_more_useful/). I ran a quick script to replace the power poles' hidden fluid components and the result are as follows:
```

Reply:
>  - which is very important

Maybe elaborate a little bit on why this is very important, cause spelling it out seems like a good idea here. I found these sentences a little hard to understand to be honest, but I don't really know if that's just cause I'm tired or because this should be rewritten, just wanted to mention it anyways.

Reply:
That's the exact type of feedback I like. I'll see what I can do :)